src/exercises/assignBookmarksToExercises.js

```diff
@@ -68,27 +69,38 @@ function popNElementsFromList(l, n) {
   return a;
 }
 
-function assignBookmarksWithLearningCycle(bookmarks, exerciseTypesList) {
-  function _removeExerciseFromList(exercise, list) {
-    return list.filter((ex) => ex !== exercise);
-  }
+function removeExerciseFromList(exercise, list) {
```

Comment:
Couldn't this just be a generic utility function `removeItemFromList` which takes an item and a list?